packages/sc-project-initializer/commands/init/README.md

```diff
@@ -2,11 +2,7 @@
 
 You now have your own AssemblyScript project setup, with Massa's sdk installed.
 
-You can now run `npm run build` to compile your AssemblyScript files.
-
-By default it will build all files in `assembly/contracts` directory.
-
-To use libraries as `massa-as-sdk` and `@massalabs/as` you need to import the required function, for instance :
+To use libraries as `@massalabs/massa-as-sdk` and `@massalabs/as` you need to import the required function, for instance :
```

Comment:
To use libraries, like `@massalabs/massa-as-sdk` and `@massalabs/as`, you need to import the required function, for instance :

Reply:
Just a comment: isn't it possible to import all functions at once, not one by one? If yes and if it's not trivial to devs to know this, maybe we should mention it?

Reply:
For some reason, it has been splitted in two separate repos `@massalabs/massa-as-sdk`  `@massalabs/as`. thats mean you need to import both separately. a solution to make it simple would be to reexport stuffs from "as" in "as-sdk", thus only importing the sdk would work